init.md

```diff
@@ -33,9 +33,19 @@ Terminology
 -----------
 
 - The `.git` folder is referred to as the "repository" or
-  "local repository"
+  **"local repository"**
 - The directory above the `.git` folder, where you edit your
-  actual files, is called the "working directory"
+  actual files, is called the "working directory" or **"workspace"**
+- Refer to [Git Lifecycle diagram](images/git-transport.png)
+
+
+Summary of commands
+-------
+
+```sh
+mkdir -p test; cd test;
```

Comment:
Love all the changes here. Not sure about this summary though, how do you see this section being used?

Reply:
Some users may want to just want to quickly progress through. Providing a single snippet of code they can copy/paste into the terminal provides that convenience.